### ISSUE TITLE
Polarized Fractal Efficiency indicator

### ISSUE THREAD

**Issue by jayMcBee** [TulipCharts/tulipindicators#107]
>  Polarized Fractal Efficiency (PFE) is a technical indicator that was developed by Hans Hannula to determine price efficiency over a user-defined period. This indicator fluctuates between -100 and +100, with 0 as the centerline. Securities with a PFE greater than zero are deemed to be trending up, while a reading of less than zero indicates the trend is down. Polarized Fractal Efficiency's signature characteristic is its use of fractal geometry in determining how efficiently a security's price is moving. 

A description of the formula is available e.g. here
https://www.investopedia.com/terms/p/pfe.asp

**Comment by codeplea** [comment]
Do you have a primary source? And example calculations?

**Comment by kevindirect** [comment]
Has there already been work on this? #68

**Comment by codeplea** [comment]
It's in the beta directory: https://github.com/TulipCharts/tulipindicators/blob/master/beta/pfe.c

It's likely buggy.

**Comment by kevindirect** [comment]
Yup, I read up on the beta indicators in the 9.1 release issue #114 after I found that pfe was merged.
Tangent: it would be interesting to note what needs to be changed/improved/tested for each of these beta implementations if it is known. That way we could know how close each one is to being able to be mainlined. Perhaps a roadmap issue for beta implementations and new requested indicators.
